backend/corpora/dataset_processing/process.py

```diff
@@ -207,6 +207,10 @@ def main():
         elif step_name == "cxg_remaster":
             from backend.corpora.dataset_processing.remaster_cxg import process
 
+            process(dataset_id, os.environ["CELLXGENE_BUCKET"], dry_run=False)
+        elif step_name == "cxg_remaster_v2":
+            from backend.corpora.dataset_processing.remaster_cxg_v2 import process
```

Comment:
do we still need to maintain `remaster_cxg(_v1)`, above?

Reply:
probably not! i can remove